spring-ai-milestone/src/main/java/com/baeldung/spring/ai/ollamachatbot/model/HelpDeskRequest.java

```diff
@@ -0,0 +1,22 @@
+package com.baeldung.spring.ai.ollamachatbot.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
```

Comment:
We can skip the use of Lombok. We are using it here for just a constructor and it's not used for any other object of the model, so I think we can go ahead without this dependency.